src/commands/hexpire.json

```
@@ -0,0 +1,118 @@
+{
+    "HEXPIRE": {
+        "summary": "Set expiry time on hash fields.",
+        "complexity": "O(1) for each field assigned with TTL, so O(N) to add N TTLs when the command is called with multiple fields.",
```

it seems redundant mentioning both O(1) and O(N)
```suggestion
        "complexity": "O(N) where N is the number of specified fields.",
```